sd-card/config/config.ini

```diff
@@ -103,6 +103,10 @@ HomeassistantDiscovery = false
 ;Token = undefined
 ;main.Fieldname = undefined
 
+;[Webhook]
+;Uri = undefined
+;ApiKey = undefined
+

```

Comment:
Please add the image parameter
Alternatively you can give us the rights to modify your PR so we can do it.

Reply:
sry, wrong MR